imp/TokenSet.cs

```diff
@@ -175,6 +175,20 @@ private TokenSet()
             // no operation
         } /* end TokenSet */
 
+        /* ---------------------------------------------------------------------------
+        * private constructor TokenSet (uint[])
+        * ---------------------------------------------------------------------------
+        * used to instantiate the prefabricated first and follow lists
+         * ------------------------------------------------------------------------ */
+
+        public TokenSet(params uint[] TSB)
+        {
+            dataStored.segments[0] = TSB[0];
+            dataStored.segments[1] = TSB[1];
+            dataStored.segments[2] = TSB[2]; 
+            dataStored.elemCount = TSB[3];
+        }
+

```

Comment:
line 205 says `/* allocate new set */` but that is not true because `newSet` is already allocated in line 201.